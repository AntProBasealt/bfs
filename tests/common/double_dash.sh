cd basic
bfs_diff -- . -type f
