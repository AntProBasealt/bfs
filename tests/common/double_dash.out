./a
./b
./c/d
./e/f
./j/foo
./k/foo/bar
./l/foo/bar/baz
